wemake_python_styleguide/violations/best_practices.py

```diff
@@ -2020,3 +2022,44 @@ def my_function(first, /, second):
 
     error_template = 'Found positional-only argument'
     code = 451
+
+
+class ContinueInFinallyBlockViolation(ASTViolation):
+    """
+    Forbids to use ``continue`` in ``finally`` case.
+
+    Note, that we check for any ``try``, ``except`` or ``finally`` nodes.
+
+    Reasoning:
+        Putting any control statements in finally is a terrible practice.
```

Comment:
I would add that `is a terrible practice, because finally is implicitly called and can cause damage to your logic with its implicitness`.